README.md

```diff
@@ -0,0 +1,23 @@
+## Contents Of This File
+
+ * About EzContent_api
+ * Key features
+ * Drupal 8 Installation
+
+## About EzContent_api
+
+This module adds JSON API functionality to EZContent
+(https://www.drupal.org/project/ezcontent) profile. This module extends the 
```

Comment:
Make use of link instead of putting url in bracket, please see https://github.com/srijanone/ezcontent/blob/8.x/README.md for reference.